pkg/core/resources/model/resource.go

```diff
@@ -507,8 +507,7 @@ func GetNamespace(rm ResourceMeta, systemNamespace string) Namespace {
 }
 
 func ComputeLabels(
-	rd ResourceTypeDescriptor,
-	spec ResourceSpec,
+	resource Resource,
```

Comment:
it's better to keep it as `rd` and `spec`, otherwise we'd have to guard `resource.GetMeta()` to not be `nil`, see the [comment](https://github.com/kumahq/kuma/pull/11449#discussion_r1764678386)

Reply:
changed